.github/workflows/build_and_test.yml

```diff
@@ -1,6 +1,10 @@
 name: Build
 
-on: push
+on: 
+  push:
+    branches:
+      - development
```

Comment:
I wonder if we should care for `beta` branch push too, wdyt?

Reply:
Yeah that's probably a good idea to future proof it, given we will be cherry picking commits onto it and probably want a CI run through each time we do so.

Reply:
☺️ I need more help. Please always support me. thanks